charts/images.yaml

```diff
@@ -188,10 +188,6 @@ images:
   tag: "0.9.0"
 
 # Miscellaenous
-- name: busybox
```

Comment:
You have to run `make generate` to remove it also from `charts/images.go`

Reply:
Hm, actually `make generate` does not work. Please add `./charts/...` here https://github.com/gardener/gardener/blob/6cdb8dc690c2464558c228105852eb3ac7c2c05a/Makefile#L212 and then run it.

Reply:
mmmmm NO. It didn't work for me so I have to remove it directly fro the file.